haiku-apps/sync_modular/sync_modular-2.2.1b.recipe

```diff
@@ -0,0 +1,79 @@
+SUMMARY="Native, modular software synthesizer"
+DESCRIPTION="It gives you, though limited as any synth, but quite versatile possibilities for creating \
+and using new sounds.
+
+Registration:
+User name : Free registration
+Serial number : eCDiPa-kbbmPZ-zGDJuyU
```

Comment:
I might be wrong, but I think that the recipe should not provide a serial number (even it's for a free registration) but the address of an official web site where such a serial number can be obtained.